File: gpt_multiagent_python_openai_jira_social_plus/api/social.py
def handler(request):
    import json
    from datetime import datetime, timezone, timedelta
    
    headers = {
        'Content-Type': 'application/json',
        'Access-Control-Allow-Origin': '*'
    }
    
    # Timestamps pour scheduling
    now = datetime.now(timezone.utc)
    post1_time = now + timedelta(hours=2)
    post2_time = now + timedelta(hours=6) 
    post3_time = now + timedelta(days=1)
    
    response_data = {
        "campaign": "kaspa_community_showcase",
        "posts": [
            {
                "platform": "x",
                "text": "🚀 BlablaKAS révolutionne le covoiturage avec la blockchain #Kaspa ! Sécurité, transparence et communauté au cœur de chaque trajet. #BlablaKAS #KaspaCommunity #Web3",
                "media_urls": [],
                "scheduled_at": post1_time.isoformat(),
                "requires_approval": False
            },
            {
                "platform": "linkedin",
                "text": "KAScomodation transforme l'hébergement collaboratif grâce à la technologie Kaspa. Une nouvelle approche de l'économie du partage, basée sur la confiance et la décentralisation.",
                "media_urls": [],
                "scheduled_at": post2_time.isoformat(), 
                "requires_approval": False
            },
            {
                "platform": "x",
                "text": "🏠 Découvrez KAScomodation : l'hébergement entre particuliers réinventé ! Réservations sécurisées, paiements transparents, communauté Kaspa unie. #KAScomodation #Kaspa",
                "media_urls": [],
                "scheduled_at": post3_time.isoformat(),
                "requires_approval": False
            }
        ],
        "reports": [
            {
                "period": "7d",
                "kpis": ["impressions", "engagement", "ctr"]
            },
            {
                "period": "30d", 
                "kpis": ["impressions", "engagement", "subs"]
            }
        ]
    }
    
    return {
        'statusCode': 200,
        'headers': headers,
        'body': json.dumps(response_data, ensure_ascii=False)
    }

File: gpt_multiagent_python_openai_jira_social_plus/api/social.ts
import { VercelRequest, VercelResponse } from '@vercel/node';

export default function handler(req: VercelRequest, res: VercelResponse) {
  res.setHeader('Access-Control-Allow-Origin', '*');
  res.setHeader('Content-Type', 'application/json');

  const now = new Date();
  const scheduledTimes = {
    post1: new Date(now.getTime() + 2 * 60 * 60 * 1000).toISOString(),
    post2: new Date(now.getTime() + 6 * 60 * 60 * 1000).toISOString(), 
    post3: new Date(now.getTime() + 24 * 60 * 60 * 1000).toISOString()
  };

  const response = {
    campaign: "kaspa_community_showcase",
    posts: [
      {
        platform: "x",
        text: "🚀 BlablaKAS révolutionne le covoiturage avec la blockchain #Kaspa ! Sécurité, transparence et communauté au cœur de chaque trajet. #BlablaKAS #KaspaCommunity #Web3",
        media_urls: [],
        scheduled_at: scheduledTimes.post1,
        requires_approval: false
      },
      {
        platform: "linkedin", 
        text: "KAScomodation transforme l'hébergement collaboratif grâce à la technologie Kaspa. Une nouvelle approche de l'économie du partage, basée sur la confiance et la décentralisation.",
        media_urls: [],
        scheduled_at: scheduledTimes.post2,
        requires_approval: false
      },
      {
        platform: "x",
        text: "🏠 Découvrez KAScomodation : l'hébergement entre particuliers réinventé ! Réservations sécurisées, paiements transparents, communauté Kaspa unie. #KAScomodation #Kaspa",
        media_urls: [],
        scheduled_at: scheduledTimes.post3,
        requires_approval: false
      }
    ],
    reports: [
      {
        period: "7d",
        kpis: ["impressions", "engagement", "ctr"]
      },
      {
        period: "30d", 
        kpis: ["impressions", "engagement", "subs"]
      }
    ]
  };

  res.status(200).json(response);
}